CONTRIBUTING.md

```diff
@@ -108,17 +108,17 @@ The main function that does the heavy lifting (and external entry point) is `API
 
 All PRs require passing tests before the PR will be reviewed.
 
-All test files are in [`rest_test.go`](https://github.com/sendgrid/rest/blob/master/rest_test.go).
+All test files are in [`rest-test.go`](https://github.com/sendgrid/rest/blob/master/rest-test.go).
```

Comment:
This one didn't change, can you please change it back or change the file path itself?